dev/src/TestUserInput.ts

```diff
@@ -4,7 +4,7 @@
  *--------------------------------------------------------------------------------------------*/
 
 import * as assert from 'assert';
-import { Event, EventEmitter, InputBoxOptions, MessageItem, MessageOptions, OpenDialogOptions, QuickPickItem, QuickPickOptions, Uri } from 'vscode';
+import * as vscodeTypes from 'vscode'; // `TestUserInput._vscode` should be use for anything that's not purely a type (e.g. instantiating a class)
```

Comment:
nit: just a small grammar thing:
```suggestion
import * as vscodeTypes from 'vscode'; // `TestUserInput._vscode` should be used for anything that's not purely a type (e.g. instantiating a class)
```